modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -803,7 +811,7 @@
 			"DefaultFragmentRendererContext"
 		],
 		"from": "setSegmentsExperienceIds(long[] paramName)",
-		"issueKey": "LPS-198401",
+		"issueKey": "layoutItemSelectorCriterion.setShowHiddenPages(true);",
 		"to": ""
```

Comment:
@kyle-miho this doesn't seem right. Is it a typo?

Reply:
oh oops it is, fixing it real quick @nicolas-sss !

Reply:
@nicolas-sss Fixed!